docs/intro.rst

```diff
@@ -0,0 +1,60 @@
+
+.. py:currentmodule:: rticonnextdds_connector
+
+Introduction to RTI Connector
+=============================
+
+RTI Connext DDS
+~~~~~~~~~~~~~~~
+
+Connext DDS is a software connectivity framework for real-time distributed
+applications. It uses the publish-subscribe communications model to make
+data distribution efficient and robust. At its core Connext DDS is the world's
+leading ultra-high performance, distributed networking databus.
+
+RTI Connext DDS provides programming APIs in C, C++, Java, and .NET.
+
+.. note::
+
+    This documentation assumes you are already familiar with the basic DDS concepts.
+    You can learn about DDS in the *Core Libraries Getting Started Guide*,
+    *Core Libraries User's Manual*, and the *Connext DDS* API documentation for C,
+    C++, Java and .NET. These documents are available at the
+    `RTI Community portal <https://community.rti.com/documentation>`__
+
+RTI Connector
+~~~~~~~~~~~~~
+
+Connector is a family of simplified APIs that allow publishing and subscribing
+to the Connext DDS Databus in other programming languages, such as Python
+and JavaScript.
+
+In *Connector*, the DDS system is defined in XML. This includes the DDS entities
+and their data types and quality of service. Applications instantiate a
+``Connector`` object that loads an XML configuration and creates the entities
+that allow publishing and subscribing to DDS Topics.
+
+.. image:: static/overview.png
```

Comment:
Please could you create a similar image which is JavaScript-ified for me to use? If I try to replicate it it'll be slightly different